### ISSUE TITLE
3 dots -> preference menu does not always appear until mouse-over

### ISSUE THREAD

**Issue by shodanx2** [srwi/EverythingToolbar#364]
### Preflight Checklist

- [X] I have fully read the [README](https://github.com/srwi/EverythingToolbar/blob/master/README.md) and it did not solve the problem.
- [X] I have searched all [open and closed issues](https://github.com/srwi/EverythingToolbar/issues?q=) for the same bug report without success.

### EverythingToolbar Version

1.0.3.0

### Windows Version

Windows 10 19045.2604

### Steps to reproduce

Sometimes when I press the 3 dots menu and navigate to preference, the menu items do not appear

Unsure what is causing that, here is a screenshot

Uploading 2023-04-09 23-42-44.mp4…

### Expected Behavior

The menu should appear when mouse over preference menu

### Actual Behavior

Sometimes the menu appears, sometimes it requires a mouseover pass

### Log output

No output found at the relevant date stamps

**Comment by shodanx2** [comment]
It seems the screenshot didn't take, here it is again 

https://user-images.githubusercontent.com/10621885/230821806-838d8c44-3df1-4218-92d4-971f18021443.mp4

**Comment by srwi** [comment]
Does this also happen if you close all background processes? Specifically I could imagine whatever is producing those white element outlines to cause this. Could you please try disabling that and see if the problem persists after rebooting the machine?

**Comment by shodanx2** [comment]
Hello,

The white line on the systray happen because of my high contrast theme greyeves

I tried settings another, normal color theme and the issue still occurred 

![image](https://user-images.githubusercontent.com/10621885/230997214-5e62ace1-b5c5-47cd-ab72-69fc8a5811a9.png)

I thought QuickSwitch was likely the cause, however the bug still happenned with it turned off

![image](https://user-images.githubusercontent.com/10621885/230997616-d3f664b7-feb0-46a6-90b6-c3c2d7afc04f.png)

Next I turned off, gajim, synctrayzor and sharex
![image](https://user-images.githubusercontent.com/10621885/230997792-993a6a8c-d967-41b3-8629-921f317b8121.png)

Next I turned of MS Powertoys and Ditto, but still
![image](https://user-images.githubusercontent.com/10621885/230998049-f5060500-6f84-4b36-ab69-7bcfd7ee4c64.png)

Next I turned off, steam, streamdeck, greenshot and mumble
![image](https://user-images.githubusercontent.com/10621885/230998365-ad0171cf-cc98-4a2d-8415-9ff31c47399b.png)

Spacedesk, bluetooth tray, stardock groupy, eartrumper
![image](https://user-images.githubusercontent.com/10621885/230999037-b3cadb00-4603-4292-a80d-270be2047136.png)

ffmpeg, barrierd , elgato audio server, adb, bonjour
I seem to no longer be able to reproduce the issue, I bet that ffmpeg gdi screen grabber, is very cpu intensive and was making the UI skip a beat.

restarted ffmpeg, it comes back
![image](https://user-images.githubusercontent.com/10621885/230999999-3984612f-6d0c-4e27-b7c5-71c9ae7baf0e.png)

turned off ffmpeg desktop streamer again, problem goes away, it really really looks like this is the cause

ffmpeg -f gdigrab -framerate 60 -video_size 3840x2160 -i desktop -vf "scale=1920:1080" -c:v h264_nvenc -preset llhp -tune ull -f mpegts udp://239.0.0.10:9998

Just a strange UI/GDI interaction  maybe

**Comment by srwi** [comment]
I tried your command and I can't reproduce the issue while ffmpeg is recording the screen to file. Does this only happen while recording the screen?

**Comment by shodanx2** [comment]
Yes, it stopped happenning when I disabled ffmpeg. 
I am always streaming my desktop to local multicast as my screen is showing up on other monitors in the house.
The issue was intermittent, even before I started killing processes it would do it about 1/3 of the time or less.

It seems to be happenning less often as I killed more processes and completely stopped happenning when I disabled ffmpeg.
I could be a general performance issue and ffmpeg is just something that takes a lot of cpu/gpu time ?  (5700x/4070ti/64GB)

In any case, now that I know how to make the preferences menu appear, and since I have this toolbar setup to my liking and won't need that menu much, it's fixed for me. 

I was just worried about new users getting confused when nothing appears in the preference menu but even that seems to be a rare case that only affects me ?!

So I think this topic isn't really a problem anymore.
